src/ftp-deploy.js

```diff
@@ -76,8 +76,10 @@ const FtpDeployer = function() {
         this.ftp = new PromiseFtp();
 
         return this.ftp.connect(config).then(serverMessage => {
-            console.log("Connected to:", config.host);
-            console.log("Connected: Server message: " + serverMessage);
+        	if (config.verbose) {
```

Comment:
It would be more idiomatic with the rest of the code, to implement this by sending these messages out via an event, which the user could either subscribe to or ignore. That's something I should have done from the outset. If you want to change ot that let me know, otherwise I can do it